utils/make-report.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/python3
+
+# SPDX-License-Identifier: Apache-2.0
+
+##################### fixLitTest.py ########################################
```

Comment:
Thanks for this tool! I may call it from `RunONNXModel.py` sometimes in the future to show the report.

Reply:
Tx